python_modules/dagster-graphql/dagster_graphql/schema/backfill.py

```diff
@@ -101,6 +105,19 @@ class Meta:
         name = "BulkActionStatus"
 
 
+class GrapheneAssetBackfillData(graphene.ObjectType):
+    class Meta:
+        name = "AssetBackfillData"
+
+    assetPartitionsStatusCounts = non_null_list(
+        "dagster_graphql.schema.partition_sets.GrapheneAssetPartitionsStatusCounts"
+    )
+    rootAssetTargetedRanges = graphene.List(
+        graphene.NonNull("dagster_graphql.schema.partition_sets.GraphenePartitionKeyRange")
+    )
+    rootAssetTargetedPartitions = graphene.List(graphene.String)
```

Comment:
`graphene.List(graphene.NonNull(graphene.String))`

Reply:
👍

Reply:
ah actually, the intent was to only populate the ranges for time window partitions, and for other partition types to populate the set of specific partitions (since in static cases, it doesn't make as much sense to display ranges)

Reply:
let me know if you think we should populate in all cases

Reply:
The list itself would still be nullable. My suggestion is to mark the contents of the list as non nullable.

it's kind of silly but by default lists in graphene are nullable. That means the typescript type generated for this looks like:

```
rootAssetTargetedPartitions: Array<string | null> | null;
```
Notice how the contents of the array itself are nullable.

After my suggestion it will look like this:
```
rootAssetTargetedPartitions: Array<string> | null;
```

The array itself is still nullable but not the contents of the array

Reply:
makes sense!